app/javascript/techradar.js

```diff
@@ -137,7 +200,7 @@
                     })
                     .attr("r", 10)
                     .attr("class", function (d) {
-                        return (hasSpotMoved(d)) ? "spot hasMoved" : "spot";
+                        return ((hasSpotMoved(d)) ? "spot hasMoved" : "spot") + " " + quadrant.cssClass;
```

Comment:
This reasoning makes sense. Although the probability for someone else modifying it is very low.
The intent of this change actually transformed eventually. The spots are currently colored as well but that can be added also to radar. Will do those changes...